src/ripple/rpc/impl/RPCHandler.cpp

```diff
@@ -95,7 +95,7 @@ namespace {
 
      Failure:
         {
-          "error" : "noNetwork",
+          "error" : "notSynced",
           "error_code" : 16,
           "error_message" : "Not synced to Ripple network.",
```

Comment:
Let's change this message to reflect ErrorCodes.cpp  as well

Reply:
Done.